firefox-ios/Client/Frontend/Home/LegacyHomepageViewController.swift

```diff
@@ -959,6 +959,15 @@ extension LegacyHomepageViewController: HomepageViewModelDelegate {
                             category: .legacyHomepage)
         }
     }
+
+    func reloadViewIfNeeded() {
+        notificationCenter.post(name: .ShowHomepage, withUserInfo: windowUUID.userInfo)
+        notificationCenter.post(name: .HistoryUpdated)
```

Comment:
Does this mean with those changes we are not triggering the `viewWillAppear` delegate method anymore? This has impacts for telemetry events recording if it does 👀

Reply:
I understood it does, which triggered my long text above 😄